scalafix-core/shared/src/main/scala/scalafix/internal/config/Config.scala

```diff
@@ -0,0 +1,22 @@
+package scalafix.internal.config
+
+import metaconfig.ConfDecoder
+import org.langmeta._
+
+trait TargetSymbolsConfig {
```

Comment:
Could this be a case class? Is the trait meant to be extensible?

Reply:
The idea here was to keep it minimal. The only behavior we need for this type is the one specified by the methods below and the apply gives us a standard way to get it a value level. I could make it sealed.